docs/connecting-your-data/data-warehouses/connecting-to-redshift.md

```diff
@@ -72,6 +72,23 @@ You'll want to gather the following connection details from Redshift:
 
     You can also find your **Database Name** under the **Database configurations** section of the Properties tab.
 
+### (Optional) SSH Tunnel
+
+Eppo supports connecting to a Redshift cluster over an SSH tunnel.
+
+Connect with a username & password combination or public key option.
+
+**SSH tunnel with username and password**
+
+* tunnel host
+* username & password
+
+**SSH tunnel with public key**
+
+* tunnel host
+* username
+* public key
+
```

Comment:
Should we add a screenshot or some actual example? It always makes me feel better when I'm reading docs and there's a specific example.